reference/configuration/security.rst

```diff
@@ -191,8 +191,8 @@ Authentication
 ~~~~~~~~~~~~~~
 
 You can authenticate to an LDAP server using the LDAP variants of the
-``form_login`` and ``http_basic`` authentication providers. Simply use
-``form_login_ldap`` and ``http_basic_ldap``, which will attempt to
+``form_login``, ``http_basic`` and ``json_login`` authentication providers. Simply use
```

Comment:
This should be reverted and back ported in 3.4

Reply:
I don't understand this comment. The entire pull request should be done in 3.4 branch? Only part of the changes? If so, which ones? Thanks!

Reply:
Only the change on this line, adding `json_login`, it is a Symfony 3 feature.

Reply:
@HeahDude it's related to LDAP, so i think it's not related to SF3

Reply:
Ldap added its own very lately. See https://github.com/symfony/symfony/pull/18952

Reply:
@HeahDude 
yes, but the LDAP variant of json_login does not exist on Symfony 3

Reply:
Yes, and the change on this line has nothing to do with it :). There are two different changes in this PR, one must be back ported to previous version.

Reply:
```You can authenticate to an LDAP server using the LDAP variants of the``` it has to do with it.  On SF 3 there are no variant of json_login so json_login is not mention as a variant.

Reply:
Yes you're right, sorry for the confusion.